### PR TITLE
Incorporate Intellisense XML into ref-assembly nuget packages

### LINKED ISSUE #747: Add intellisense xml's to ref-pack
This issue tracks the inclusion of intellisense xmls' in our ref-assembly nuget package. 

We still need to define the following: 

- [x] Who produces the intellisense xml's
- [x] How do we consume them (checked into our repo vs. consumed as a nuget package)
- [ ] Where do we put them (presumptively, into the reference assembly packages, `Microsoft.DotNet.Wpf.GitHub`, and `Microsoft.DotNet.Wpf.DncEng`. => These will get consumed by core-sdk, unpacked and repacked into the core-sdk re-packs. 

Tracked by internal work item https://devdiv.visualstudio.com/DevDiv/_workitems/edit/896312

### PR DESCRIPTION
**Pull request by vatsan-madhavan**
Fixes #747 

See `intellisense.md` for architecture notes. 

Intellisense XML's are produced in `dotnet/dotnet-api-docs` repo. They are currently **not** published to a Nuget package or another easily consumable artifact. Thus, the process of ingestion of these XML files is a manual one at this time. 

1. From the OPS build site at https://ops.microsoft.com/#/sites/Docs/docsets/dotnet-api-docs?tabName=builds, we obtain the latest build artifacts
    - Filter by `Build Type = Intellisense`
    - Download latest package (it's a `zip` file)
2. Extract the zip contents and retain only the contents of `_intellisense\netcore-3.0` subfolder
   - Copy these contents over to a new folder hierarchy that looks like this: 
  
    ```
    DOTNET-API-DOCS_NETCOREAPP3.0-0.0.0.1-WIN32-X86
    \---_intellisense
        \---netcore-3.0
    ```
3. Compress the contents into a new zip file (typically named `dotnet-api-docs_netcoreapp3.0-0.0.0.1-win32-x86.zip`) and upload it to a blob store using Azure Storage Explorer to `netcorenativeassets` under this path: 
    - `resource-packages -> external -> windows -> dotnet-api-docs_netcoreapp3.0` 
4. During build, consume this blob, unzip and incorporate into ref-assembly nuget pakcages produced out of WPF. 

### Q&A 

**Q**.  What happens when an intellisense XML is not found? 
**A**. Nothing. We will silently ignore the problem and the build will succeed. 

**Q.**  Are there missing files today? 
**Y.** Yes, WindowsBase and PresentationBuildTasks intellisense files are missing. See https://github.com/dotnet/dotnet-api-docs/issues/2602

**Q.**. How is the zip file update? How often is it update? 
**A.** Manually. As often as needed - typically once per major milestone or preview release. 

Notes:

- This PR needs to build a new version of `Microsoft.DotNet.Arcade.Wpf` packge, and flow into to `dotnet-wpf-int`, to build a complete set of ref-assembly packages containing intellisense information. 
- There are small unrelated commits in this PR to workaround ongoing problems with the build in dotnet/wpf. These have been cherry-picked into master already.

### CODE REVIEW

File: Documentation/intellisense.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Intellisense XML Incorporation into Ref-Pack
+
+
+Intellisense XML's are produced in `dotnet/dotnet-api-docs` repo. They are currently **not** published to a Nuget package or another easily consumable artefact. Thus, the process of ingestion of these XML files is a manual one at this time.
```
Comment (rladuca) [inline Documentation/intellisense.md]: nits:  
"produced in the"
"NuGet"
"artifact"

**Comment by vatsan-madhavan** [comment]
@dotnet/wpf-developers, this is not longer "WIP" - please review

**Comment by vatsan-madhavan** [comment]
@AdamYoblick, you should take note about how this is done. The work for WinForms would be really simple. You shouldn't even have to create assets in the blob store - you can directly piggy-back onto the assets that we have created (it includes WinForms xml's), allow them to be unzipped by Arcade scripts under $(RepoRoot)\.tools\native, then merge them into your nuget package at the appropriate location. 

Later on, the only effort we'd have to do would be periodically updating the blob store with a new zip, then updating global.json and <something>.props with the new (arbitrarily chosen) version numbers.

**Comment by AdamYoblick** [comment]
Wooo thanks Vatsan :) adding @russkie for visibility (Igor is a new team member working on runtime)

**Pull request merged**